Monocle/Provider/GithubIssues/Type.dhall

```diff
@@ -0,0 +1,4 @@
+{ github_url : Optional Text
```

Comment:
Unfortunately this has the same shape as the Github provider and we'll not be able to distinguish the two when loading the YAML. So this would have to be a `github_issue_url`, but perhaps we should integrate that directly into the existing Github provider.